cluster/cluster.go

```diff
@@ -154,6 +155,7 @@ func InitClusterObject(ctx context.Context, rkeConfig *v3.RancherKubernetesEngin
 		ConfigDir:                     flags.ConfigDir,
 		StateFilePath:                 GetStateFilePath(flags.ClusterFilePath, flags.ConfigDir),
 		PrivateRegistriesMap:          make(map[string]v3.PrivateRegistry),
+		CLIFlags:                      &flags,
```

Comment:
can we just include the dind flag instead of passing all flags to the cluster object

Reply:
thats true, but for consistency then we should remove other flags like configDir and configPath from the cluster object definition, either way are correct, its up to you.